istioctl/cmd/root.go

```diff
@@ -237,9 +237,6 @@ debug and diagnose their Istio mesh.
 	experimentalCmd.AddCommand(waitCmd())
 	experimentalCmd.AddCommand(mesh.UninstallCmd(loggingOptions))
 	experimentalCmd.AddCommand(configCmd())
-	postInstallWebhookCmd := Webhook()
-	deprecate(postInstallWebhookCmd)
-	postInstallCmd.AddCommand(postInstallWebhookCmd)
```

Comment:
Can you check whether postInstallCmd is only for postInstallWebhookCmd? If so, after postInstallWebhookCmd is removed, postInstallCmd can be removed too.

Reply:
yes, it looks to me, post-install command only have subcommands: webhook related commands. so it's safe to remove post-install command

Reply:
or we have other subcommands want to add under post-install? needs @esnible and @howardjohn 's inputs

Reply:
@carolynhu Thanks for checking that! If other reviewers are ok with removing the post-install command,  please remove it since it only has the webhook subcommand.

Reply:
yes, checked with @esnible , we should remove it